### DISCUSSION TITLE
OpenBooks

### DISCUSSION THREAD

**Discussion by hxck** [selfhst/icons#90] [New Icon Request]
Icon for https://github.com/evan-buss/openbooks

**Comment by kusold** [comment]
OpenBooks doesn't really have a logo per-say. They are using Font Awesome's `open-book`: https://github.com/evan-buss/openbooks/blob/c9273ef98a46698130f80c6577baf250dca5665c/docs/mkdocs.yml#L34

FA's SVG: https://fontawesome.com/icons/book-open?f=classic&s=solid

**Comment by shollyethan** [comment]
Thanks for the link! OpenBooks has been added using the FA SVG.
